src/app/static/src/app/components/adr/ADRInputDataset.vue

```diff
@@ -69,6 +69,7 @@ interface Data {
         newDatasetReleaseId: string | null;
         pollingId: number | null;
         modelOptions: number;
+        AdrDatasetType: any;
```

Comment:
i believe you want this
```suggestion
        AdrDatasetType: typeof AdrDatasetType;
```